velox/dwio/parquet/writer/arrow/tests/MetadataTest.cpp

```diff
@@ -125,6 +127,19 @@ void AssertEncodings(
   ASSERT_EQ(encodings, expected);
 }
 
+static std::string createFile(
+    std::shared_ptr<exec::test::TempFilePath> temp_file_path,
+    std::shared_ptr<arrow::Buffer> buffer) {
+  auto file_path = temp_file_path->getPath();
```

Comment:
Let's use the Velox coding conventions for the code we modify.
Should be `filePath`.